doc/adr/0009-container-image-tags.md

```diff
@@ -0,0 +1,52 @@
+# 9. Convention for tagging of container images
+
+Date: 2020-03-13
+
+## Status
+
+Proposed
+
+## Context
+
+Libero's products are assembled from product specific, general libero and external container images.
+
+Integration testing and reproducible deploys require uniquely identifiable image tags.
+
+As a starting point for users each product has an umbrella repo. This is not only used to collect product issues and documents. I also contains e.g. docker-compose files, helm charts that combine various components to a useable whole.
+
+
+## Decision
+
+### Tagging 
+
+- libero container images shall always be tagged with {branch-full_git_hash}
+- images shall receive semver tags when the git repo receives them
+- a semver tagging event bumps its parent tags
+
+Examples:
+
+- liberoadmin/reviewer-client:master-full_git_hash
+- liberoadmin/reviewer-client:dev-full_git_hash
```

Comment:
`develop` has been an eLife convention, it is an example that people may expect to see. Not saying projects now have to use `develop`, separate decision.